sirepo/package_data/static/js/cloudmc.js

```diff
@@ -2076,14 +2129,45 @@ SIREPO.viewLogic('tallyView', function(appState, panelState, $scope) {
         inds.forEach(i => {
             panelState.showEnum('filter', '_type', type(i), true, i - 1);
         });
+        
+    }
+    
+    function validateEnergyFilter(filter) {
+        if (! filter) {
+            return;
+        }
+        const rangeFields = ['start', 'stop'];
+        if (rangeFields.map(x => filter[x]).some(x => x == null)) {
+            return;
+        }
+        const isValid = filter.start < filter.stop;
+        rangeFields.forEach(x => {
+            validationService.validateField(
+                'energyFilter',
+                x,
+                'input',
+                isValid,
+                'Start must be less than stop'
+            );
+        });
```

Comment:
Is there a way to test this error message in the UI? I didn't see a way to get it in an invalid state.

Reply:
You should get the message by setting the start value to be greater than the stop value:
<img width="479" alt="Screenshot 2023-12-18 at 10 11 45" src="https://github.com/radiasoft/sirepo/assets/30352290/e6efcab9-df8b-4589-b8ff-7e4003feba06">

Reply:
Nice, I thought it was somehow associated with the JRange.